pkg/ami/marketplace/assets/server.conf

```diff
@@ -398,23 +493,34 @@ line.tcp.auth.db.path=conf/auth.txt
 
 ################ PG Wire settings ##################
 
-pg.enabled=true
-#pg.net.active.connection.limit=10
+#pg.enabled=true
 #pg.net.bind.to=0.0.0.0:8812
-#pg.net.event.capacity=1024
-#pg.net.io.queue.capacity=1024)
-#pg.net.idle.timeout=300000
+#pg.net.connection.limit=10
+# Windows OS might have a limit on TCP backlog size. Typically Windows 10 has max of 200. This
+# means that even if active.connection.limit is set over 200 it wont be possible to have this many
+# concurrent connections. To overcome this limitation Windows has an unreliable hack, which you can
+# exercise with this flag. Do only set it if you actively try to overcome limit you have already
+# experienced. Read more about SOMAXCONN_HINT here https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-listen
+#pg.net.connection.hint=false
+
+#pg.net.connection.timeout=300000
+
 #Amount of time in ms a connection can wait in the listen backlog queue before its refused. Connections will be aggressively removed from the backlog until the active connection limit is breached
-#pg.net.queued.timeout=300000
-#pg.net.interest.queue.capacity=1024
-#pg.net.listen.backlog=50000
-#pg.net.recv.buf.size=-1
-#pg.net.send.buf.size=-1
+#pg.net.connection.queue.timeout=300000
+
+# SO_RCVBUF value, -1 = OS default
+#pg.net.connection.rcvbuf=-1
+
+# SO_SNDBUF value, -1 = OS default
+#pg.net.connection.sndbuf=-1
+
 #pg.character.store.capacity=4096
 #pg.character.store.pool.capacity=64
 #pg.connection.pool.capacity=64
```

Comment:
Doesn't 1-per-boot.sh rely on pg.password=PG_PASSWORD_REPLACE  ?

Reply:
Good spot. Restored all original prop values in 4fe2eb84fb9bbc868fad01e3404b921ec07d6e7f